standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/RetryingMetaStoreClient.java

```diff
@@ -226,6 +239,11 @@ public Object run() throws MetaException {
         Throwable t = e.getCause();
         if (t instanceof TApplicationException) {
           TApplicationException tae = (TApplicationException)t;
+          if(tae.getMessage().contains("Internal error processing")){
```

Comment:
I think we should not retry any `InvocationTargetException` - or do we have some which should be retried?

or probably you may want to add `TApplicationException.INTERNAL_ERROR` to the `switch` below?